gensim/test/test_word2vec.py

```diff
@@ -461,6 +462,141 @@ def test_cbow_neg(self):
                                   min_count=5, iter=10, workers=2, sample=0)
         self.model_sanity(model)
 
+    def test_most_similar_among_CBOW(self):
+        """
+        Testing most_similar_among method for CBOW model
+        """
+
+        #Testing Error
+        model = word2vec.Word2Vec(sentences, size=2, sg=0, min_count=1, hs=1, negative=0)
+        self.assertRaises(ValueError, model.wv.most_similar_among, positive=['graph'])
+
+        res_voc = model.wv.index2word[:5] #Gives first 5 words in model vocab
+        #Testing Warnings
```

Comment:
logging is used. Suppress warnings becomes useful in case `words-list` is generated on the go and the user does not want to know about missing words.